src/services/users/coach.hooks.js

```diff
@@ -23,6 +24,10 @@ module.exports = {
         rec.coach = rec.coach || {};
         rec.coach.is = true;
       }),
+      iff(context => context.params.user.manager.is,
+        alterItems((rec, context) => {
+          rec.region = context.params.user.region;
```

Comment:
Region might not exist, does this handle it?

Reply:
It should be a requirement for a manager to have a region, but I can still add a handler to this if you want

Reply:
Make it throw an error if the user manager does not have a region
That will solve most things